runtime/calamari/src/fee.rs

```diff
@@ -125,133 +65,123 @@ mod multiplier_tests {
+                // which is additional super powers for the attacker
+                let fee = TransactionPayment::compute_fee(len, &info, 0);
+                fees_paid += fee;
+
+                // this will update the multiplier.
+                System::set_block_consumed_resources(block_weight, len.try_into().unwrap());
+                TransactionPayment::on_finalize(1);
+                let next = TransactionPayment::next_fee_multiplier();
+
+                assert!(next > multiplier, "{:?} !>= {:?}", next, multiplier);
+                multiplier = next;
 
-                // base_fee and len_fee are not adjusted:
-                // https://docs.substrate.io/main-docs/build/tx-weights-fees/#:~:text=A%20closer%20look%20at%20the%20inclusion%20fee
-                let adjusted_fee = fee_adjustment.saturating_mul_acc_int(fee) + base_fee + len_fee;
-                accumulated_fee += adjusted_fee;
                 println!(
-                    "Iteration {}, New fee_adjustment = {:?}. Adjusted Fee: {} KMA, Total Fee: {} KMA, Dollar Value: {}",
-                    iteration,
-                    fee_adjustment,
-                    adjusted_fee / KMA,
-                    accumulated_fee / KMA,
-                    (accumulated_fee / KMA) as f32 * kma_price,
+                    "block = {} / multiplier {:?} / fee = {:?} / fees so far {:?} / fees so far in USD {:?}",
+                    blocks, multiplier, fee, fees_paid, (fees_paid as f32 / KMA as f32) * kma_price
                 );
-            }
 
-            if accumulated_fee < target_daily_congestion_cost_kma {
-                panic!("The cost to fully congest our network should be over the target_daily_congestion_cost_kma after 1 day.");
-            }
-        });
-    }
+                if blocks == 7200 {
+                    fees_after_one_day = fees_paid;
+                    if fees_paid < target_daily_congestion_cost_kma {
+                        should_fail = true;
+                    }
+                }
+            });
+            blocks += 1;
+        }
 
-    #[test]
-    fn multiplier_can_grow_from_zero() {
-        let minimum_multiplier = MinimumMultiplier::get();
-        let target = TargetBlockFullness::get()
-            * BlockWeights::get()
-                .get(DispatchClass::Normal)
-                .max_total
-                .unwrap();
-        // if the min is too small, then this will not change, and we are doomed forever.
-        // the weight is 1/100th bigger than target.
-        run_with_system_weight(target * 101 / 100, || {
-            let next = SlowAdjustingFeeUpdate::<Runtime>::convert(minimum_multiplier);
-            assert!(
-                next > minimum_multiplier,
-                "{next:?} !>= {minimum_multiplier:?}"
-            );
-        })
+        println!(
+            "It will take {:?} days to reach multiplier of 1",
+            (blocks * 12) as f32 / (60 * 60 * 24) as f32,
```

Comment:
Parametrize the block time in all tests, otherwise this will break with async backing